cmd/flags.go

```diff
@@ -206,3 +209,16 @@ func EnvOrDefault(key string, defaultVal string) string {
 	}
 	return defaultVal
 }
+
+func PrepareOSEnv() error {
+	if runtime.GOOS != "windows" {
+		return nil
+	}
+
+	regEnvMap, err := env.WindowsRegistryEnvMap()
+	if err != nil {
+		return err
+	}
+
+	return env.PrepareWindowsOSEnv(regEnvMap, os.Getenv, os.Setenv)
+}
```

Comment:
I feel like an a os-agnostic name `PrepareOSEnv` is better for `main` but wanted to keep windows-specific names below that.

I split up windows-build only `WindowsRegistryEnvMap` from the business logic unit `PrepareWindowsOSEnv` so unit tests for the latter could run on any OS.